clients/privacy-center/cypress/e2e/consent-banner-tcf.cy.ts

```diff
@@ -1237,21 +1241,26 @@ describe("Fides-js TCF", () => {
     });
   });
 
-  describe("User preference sources of truth for UI and CMP API", () => {
+  /**
+   * There are the following potential sources of user preferences:
+   * 1) fides_string override option (via config.options.fidesString)
+   * 2) DEFER: preferences API (via a custom function)
+   * 3) local cookie (via fides_consent cookie)
+   * 4) "prefetched" experience (via config.options.experience)
+   * 5) experience API (via GET /privacy-experience)
+   *
+   * These specs test various combinations of those sources of truth and ensure
+   * that Fides loads the correct preferences in each case.
+   */
+  describe("user preferences overrides", () => {
     beforeEach(() => {
       cy.getCookie(CONSENT_COOKIE_NAME).should("not.exist");
     });
-    it("prefers preferences from a cookie when both cookie and experience exist", () => {
```

Comment:
The diff is confusing here but what I did was hoist the `cy.setCookie` logic into a shared helper function and move that above the first spec.

Reply:
thanks! We do have a ticket to further refactor / extract out common code in this test suite, so this helps towards that goal- https://github.com/ethyca/fides/issues/4303